docs/design.md

```diff
@@ -195,6 +195,14 @@ Local paths are currently prioritised over other forms. It is presumed
 - Index offsets need to be relative to the CARv1, and not absolute in the
   physical CARv2 file.
 - Index needs to be iterable.
+- Given a CARv2 file, we should be able to decompose it to the corresponding Carv1 payload and the Index.
```

Comment:
what form of decomposition do you need? is a `ReaderAt` providing streaming or random access to the Carv1 payload sufficient, or do you want to realize the carv1 payload on disk?

Reply:
I think a `ReadSeekCloser` + `ReaderAt` would be most flexible.